lib/msal-common/src/error/ClientAuthError.ts

```diff
@@ -48,6 +49,18 @@ export const ClientAuthErrorMessage = {
         code: "state_mismatch",
         desc: "State mismatch error. Please check your network. Continued requests may cause cache overflow."
     },
+    nonceMismatchError: {
+        code: "nonce_mismatch",
+        desc: "Nonce mismatch error. Please check whether concurrent requests are causing this issue."
```

Comment:
This doesn't seem as helpful as it could be. We should support concurrent requests without issue, is there some particular  nuance this makes this happen?

Reply:
"May be caused by a race condition concurrent requests"

Reply:
Fixed.